### PR TITLE
feat: [CCM-8037]: RBAC changes in CCM module

### PR DESCRIPTION
**Pull request by jyotiarora-harness**
### Summary

RBAC changes in CCM module

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 254 Issues 
* Code issue 254 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10328)

File: src/modules/75-ce/components/COAccessPointList/COAccessPointList.tsx [hunk 1]
```diff
@@ -248,6 +267,27 @@ const COLoadBalancerList: React.FC = () => {
...
+        <PageSpinner />
+      </div>
+    )
+  }
+
+  if (error) {
+    if (error.status === 403) {
+      return (
+        <Container className={css.loaderContainer}>
+          <HandleError imgSrc={PermissionError} errorMsg={get(error, 'data.message', '')} />
```
Comment (rajarshee-harness) [inline src/modules/75-ce/components/COAccessPointList/COAccessPointList.tsx:283]: You can use `getErrorInfoFromErrorObject`

File: src/modules/75-ce/components/PermissionError/__tests__/PermissionError.test.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const params = {
+  accountId: 'TEST_ACC'
+}
+
+describe('Test cases for permission error component', () => {
+  test('Should render the permission error', () => {
+    const { container } = render(
+      <TestWrapper pathParams={params}>
+        <HandleError imgSrc={PermissionError} errorMsg={'Missing permission'} />
```
Comment (rajarshee-harness) [inline src/modules/75-ce/components/PermissionError/__tests__/PermissionError.test.tsx:15]: Move to `strings.en.yaml`

File: src/modules/75-ce/components/COAccessPointList/COAccessPointList.tsx (hunk 1 above)
Comment (rishik30) [reply src/modules/75-ce/components/COAccessPointList/COAccessPointList.tsx:283]: we can't use the generic method for fetching errors as the lightwing service is written in go and has a different response structure compared to other services written in Java. So to take out the error message, I need to handle it explicitly.

File: src/modules/75-ce/pages/business-mapping/BusinessMapping.tsx [hunk 3]
```diff
@@ -104,6 +128,15 @@ const BusinessMappingPage: () => React.ReactElement = () => {
             subtitle={getString('ce.businessMapping.emptySubtitles')}
             buttonText={getString('ce.businessMapping.newButton')}
             buttonAction={() => setDrawerOpen(true)}
+            isBtnDisabled={!canEdit}
+            buttonTooltip={
+              !canEdit ? (
+                <RBACTooltip
```
Comment (jyotiarora-harness) [inline src/modules/75-ce/pages/business-mapping/BusinessMapping.tsx:134]: Use it from getToolTip function to make it consistent

**Review by jenil-harness — APPROVED** [review]

**Review by rishik30 — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by rishik30 — APPROVED** [review]

**Review by jenil-harness — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**